tests/clone_test.go

```diff
@@ -277,9 +277,9 @@ var _ = Describe("VirtualMachineClone Tests", Serial, func() {
 				libvmi.WithLabels(vmi.Labels))
 
 			By(fmt.Sprintf("Creating VM %s", vm.Name))
-			vm, err := virtClient.VirtualMachine(vm.Namespace).Create(context.Background(), vm, v1.CreateOptions{})
+			updatedVm, err := virtClient.VirtualMachine(vm.Namespace).Create(context.Background(), vm, v1.CreateOptions{})
 			Expect(err).ShouldNot(HaveOccurred())
-			return
+			return updatedVm
```

Comment:
I would remove this change and re-use the `vm` variable. Not sure how having two variables is less confusing or more readable.

Reply:
Ah, this code was removed by another commit.

Can you please squash some commits? 7 commits are a lot for such a small change and commits are not additive. I've easily reviewed this by file differences, then when re-reviewed by commits it was more confusing.

Reply:
mm, mabey we're talking about different things, but if you're talking about the change of variable `vm` to `updatedVm`  - then it's not removed (just moved :) ) . 
The reason I do this, is to make it clear that the object returned from the Create function is different than the one sent. 
However it is not a must in this case, so if you prefer I will remove it

Reply:
> mm, mabey we're talking about different things, but if you're talking about the change of variable vm to updatedVm - then it's not removed (just moved :) ) .

Thanks for clarifying :+1: 
I still think it'd be easy to review with much less commits :)

> The reason I do this, is to make it clear that the object returned from the Create function is different than the one sent.
> However it is not a must in this case, so if you prefer I will remove it

Don't feel strongly about it, just don't think it's particularly beneficial and a bit harder to read. Your call :)